src/Rules/Methods/AlwaysUsedMethodExtension.php

```diff
@@ -0,0 +1,12 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Methods;
+
+use PHPStan\Reflection\MethodReflection;
+
+interface AlwaysUsedMethodExtension
```

Comment:
This needs `@api` and also description similar to https://apiref.phpstan.org/1.11.x/source-src.PhpDoc.StubFilesExtension.html#5.